Userland/Libraries/LibJS/AST.h

```diff
@@ -192,19 +192,19 @@ class ExpressionStatement final : public Statement {
 };
 
 template<typename Func, typename... Args>
-concept ThrowCompletionOrVoidFunction = requires(Func func, Args... args)
-{
-    {
-        func(args...)
-        }
-        -> SameAs<ThrowCompletionOr<void>>;
-};
+concept ThrowCompletionOrVoidFunction = requires(Func func, Args... args) {
+                                            {
+                                                func(args...)
+                                                }
+                                                -> SameAs<ThrowCompletionOr<void>>;
+                                        };
```

Comment:
This is a disaster,  but oddly the clang-format-14 version was a disaster as well. So meh.

Reply:
(will be fixed in clang-format-16, nothing to do for now)